canned/consul.go

```diff
@@ -22,8 +23,10 @@ type Consul struct {
 
 func NewConsul(ctx context.Context) (*Consul, error) {
 	os.Setenv("TC_HOST", "localhost")
+	skipReaper, _ := strconv.ParseBool(os.Getenv("TESTCONTAINERS_RYUK_DISABLED"))
```

Comment:
@amit-shekhar @lovlin-thakkar - How should we handle invalid value (non-bool) passed for TESTCONTAINERS_RYUK_DISABLED?

Reply:
anyone setting a non-bool trying to set true will notice it is not working (default as false) & will understand that they've set the value wrongly. 
if they're setting non-bool trying to set false, it'll work for them.